mscore/events.cpp

```diff
@@ -729,8 +729,16 @@ void ScoreView::mouseDoubleClickEvent(QMouseEvent* mouseEvent)
 
       Element* clickedElement = elementNear(toLogical(mouseEvent->pos()));
 
-      if (!(clickedElement && clickedElement->isEditable()))
+      if (!clickedElement)
             return;
+      else if (!clickedElement->isEditable()) {
+            if (clickedElement->isText()
+                && (static_cast<Text*>(clickedElement)->tid() == Tid::FOOTER || static_cast<Text*>(clickedElement)->tid() == Tid::HEADER)) { // double-click an instrument name to open the edit staff/part properties menu
```

Comment:
You can use `toText(clickedElement)` as a shortcut for the `static_cast`

Reply:
Nice!